microdata_tools/validation/steps/dataset_validator.py

```diff
@@ -49,7 +49,7 @@ def _valid_value_column_check(
         unique_codes = list(
             set(code_list_item["code"] for code_list_item in code_list)
         )
-        if sentinel_list is not None:
+        if sentinel_list:
```

Comment:
I see we remove the check for None, can it still be none since we have?  `sentinel_list: Union[List, None] `

Reply:
The code still check for None. “if sentinel list:” will evaluate to True if the sentinel list is a non-empty list (and not None), else it will evaluate to false 👍🏻

Reply:
Great!